db/queries/assessment_records/queries.py

```diff
@@ -270,3 +272,22 @@ def get_application_jsonb_blob(application_id: str) -> dict:
     application_jsonb_blob = db.session.scalar(stmt)
     application_json = AssessorTaskListMetadata().dump(application_jsonb_blob)
     return application_json
+
+
+def bulk_update_location_jsonb_blob(application_ids_to_location_data):
+    stmt = (
+        update(AssessmentRecord)
+        .where(AssessmentRecord.application_id == bindparam("app_id"))
+        .values(location_json_blob=bindparam("new_location_data"))
+    )
+
+    update_params = [
+        {
+            "app_id": item["application_id"],
+            "new_location_data": item["location"],
```

Comment:
I guess it's a bit late as we've done the migration already? But can we avoid temporal references if possible? So `location_data` would be preferable to `new_location_data`.